### DISCUSSION TITLE
/ping is returning 404

### DISCUSSION THREAD

**Discussion by jelbatnigi** [gotenberg/gotenberg#592] [Q&A] [answered]
I installed the version 7. I keep getting 404 when calling the /ping endpoint. any idea why?

{"level":"error","ts":1674226562.0985687,"logger":"api","msg":"code=404, message=Not Found","trace":"47faa6c5-530e-42ab-b046-4121be0477e4","remote_ip":"172.17.0.1","host":"localhost:3000","uri":"/ping","method":"GET","path":"/ping","referer":"","user_agent":"PostmanRuntime/7.29.2","status":404,"latency":182800,"latency_human":"182.8µs","bytes_in":0,"bytes_out":9}

**Comment by NiklasBr** [comment, marked as answer]
It should be `/health` in 7.x

**Reply by jelbatnigi** [reply]
Thanks

**Reply by gulien** [reply]
Thanks @NiklasBr
